packages/@dcl/ecs/package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@dcl/crdt": "file:../crdt",
     "@dcl/js-runtime": "file:../js-runtime",
-    "@dcl/protocol": "1.0.0-4085628047.commit-0f6384e"
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-4105728390.commit-dfd5b79.tgz"
```

Comment:
Merge first https://github.com/decentraland/protocol/pull/109, and then update here with `npm i @dcl/protocol@next`